### ISSUE TITLE
Going fullscreen on click using Firefox loses all control functionality

### ISSUE THREAD

**Issue by ghost** [metalx1000/Phaser_Game_Template#1]
[Visiting your live version](http://filmsbykris.com/scripts/Phaser_Game_Template/) while using Firefox 56 (the only browser I've tested on this) shows your game filling the browser window but not yet in full screen mode. Clicking anywhere on the game makes the browser go into full screen and I lose all ability to click on menu items and sprites.

**Comment by metalx1000** [comment]
Yeah, I need to update this.  The project is using an older version of Phaser and things have changed.  Once I make some changes and get it working with the current version of Phaser it should work again (fingers crossed).
